server/routes/userRouter.js

```diff
@@ -0,0 +1,11 @@
+const userRouter = require('express').Router();
+const {getUser, updateUser, deleteUser} = require('../controllers/users');
+
+userRouter.route('/login').post(getUser);
+
+userRouter.route('/:users')
+.get(getUsers)
```

Comment:
For the '/:users', it should say '/:userId' and have the following: 
`get(getUser), put(updateUser), delete(deleteUser)`

Reply:
We pushed new changes. Please let me know if you have any quesetions.